docs/changes/0.35.0beta1.rst

```diff
@@ -0,0 +1,56 @@
+QCoDeS 0.35.0beta1 (2022-07-08)
```

Comment:
beta1? not b1?

Reply:
I was sure its beta but you are right (see pep440)